src/se3/so3.cpp

```diff
@@ -0,0 +1,14 @@
+#include "se3.hpp"
+
+SO3::SO3(Eigen::Quaterniond const& quaternion) : mQuaternion(quaternion) {
+}
```

Comment:
Throw an exception it given Quaternion is not normalized as that is an invariant of SO3